### PR TITLE
Add new branding docs

### PR DESCRIPTION
**Pull request by Yoshani**
## Related Issues
- Issue https://github.com/wso2/product-is/issues/15729

## Related PRs
- PR https://github.com/wso2/identity-apps/pull/3931
- PR https://github.com/wso2/identity-apps/pull/3943
- PR https://github.com/wso2/identity-apps/pull/3951
- PR https://github.com/wso2/identity-apps/pull/3956

### CODE REVIEW

File: en/docs/references/extend/myaccount/customize-my-account-ui.md [hunk 1]
```diff
@@ -14,212 +14,53 @@ For information on the Semantic UI theming, see [Semantic UI documentation](http
...
-    window["runConfig"] = {
-        ...
-        applicationName: "NIGHT OWL EXPRESS",
-        copyrightText: "Night Owl Express © 2020"
-    };
-    ```
-    
-6.  Replace `favicon.ico` in the `<IS_HOME>/repository/deployment/server/webapps/my-account` folder with the
-    desired icon.
+To re-brand different tenants, create folders for each tenant named after the tenant domain, inside the newly created `branding` folders.
```
Comment (divyaamunugama) [inline en/docs/references/extend/myaccount/customize-my-account-ui.md:27]:
```suggestion
To rebrand different tenants, create folders for each tenant named after the tenant domain, inside the newly created `branding` folders.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 2]
```diff
@@ -65,74 +64,111 @@ To customize these pages, follow the steps below.
...
     - `assets` folder
 
 !!! note
-    Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+    Repeat the above steps for `accountrecoveryendpoint` as well.
 
-### Step 3: Add a custom layout design
+### Step 3: Add the branding-preference.json file
+
+The branding-preference.json file is used to specify which layout should be used. To add this file, follow the guide [Re-brand WSO2 Identity Server UIs]({{base_path}}/references/extend/rebranding/rebrand-identity-server-uis/).
```
Comment (divyaamunugama) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:101]:
```suggestion
The branding-preference.json file is used to specify which layout should be used. To add this file, follow the guide [Rebrand WSO2 Identity Server UIs]({{base_path}}/references/extend/rebranding/rebrand-identity-server-uis/).
```

File: en/docs/references/extend/myaccount/customize-my-account-ui.md [hunk 3]
```diff
@@ -14,212 +14,53 @@ For information on the Semantic UI theming, see [Semantic UI documentation](http
...
-    ```java
-    $ git fetch --all --tags --prune
-    $ git checkout tags/v1.4.28 -b feature-dark-theme-demo
-    ```
-
-3. Navigate to the `modules/theme/src/themes` folder within identity-apps. All the theme global variable overrides can be found in the `modules/theme/src/themes/default/globals/site.variables` file. For the full set of variables, see the [original theme variables file](https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/themes/default/globals/site.variables).
-
----
-
 Follow the steps given below to further customize **My Account** application.
```
Comment (divyaamunugama) [inline en/docs/references/extend/myaccount/customize-my-account-ui.md:17]:
```suggestion
Follow the steps given below to further customize the **My Account** application. 
```

File: en/docs/references/extend/myaccount/customize-my-account-ui.md (hunk 1 above)
Comment (divyaamunugama) [inline en/docs/references/extend/myaccount/customize-my-account-ui.md:27]:
```suggestion
To rebrand different tenants, create folders for each tenant named after the tenant domain, inside the newly created `branding` folders.
```

File: en/docs/references/extend/myaccount/customize-my-account-ui.md [hunk 4]
```diff
@@ -14,212 +14,53 @@ For information on the Semantic UI theming, see [Semantic UI documentation](http
...
-        ...
-        applicationName: "NIGHT OWL EXPRESS",
-        copyrightText: "Night Owl Express © 2020"
-    };
-    ```
-    
-6.  Replace `favicon.ico` in the `<IS_HOME>/repository/deployment/server/webapps/my-account` folder with the
-    desired icon.
+To re-brand different tenants, create folders for each tenant named after the tenant domain, inside the newly created `branding` folders.
+For example, if the tenant domain is `rusticfox.com`, create a folder named `rusticfox.com` inside the `extensions/branding` folder.
```
Comment (divyaamunugama) [inline en/docs/references/extend/myaccount/customize-my-account-ui.md:28]:
```suggestion
For example, if the tenant domain is `rusticfox.com`, create a folder named `rusticfox.com` in the `extensions/branding` folder.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
-# Customize Login Pages
+# Customize Login Page Layouts
 
 You can customize the layout and branding of user interfaces like the login page, register page, username and password recovery pages, and single sign-on pages.
```
Comment (divyaamunugama) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:3]:
```suggestion
You can customize the layout and branding of user interfaces like the login page, register page, username, and password recovery pages, and single sign-on pages.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 
-All UIs are available in the `authenticationendpoint`, `accountrecoveryendpoint`, and `x509certificateauthenticationendpoint` apps located in the `<IS_HOME>/repository/deployment/server/webapps` folder.
+All UIs are available in the `authenticationendpoint` and `accountrecoveryendpoint` apps located in the `<IS_HOME>/repository/deployment/server/webapps` folder.
 
 ## General components of the UI
 
-All pages of WSO2 Identity Server are separated into three general components as shown below. You can reference these components when you write the custom layout using a special notation.
+All pages of WSO2 Identity Server are separated into two general components as shown below. You can reference these components when you write the custom layout using a special notation.
```
Comment (divyaamunugama) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:9]:
```suggestion
All pages of the WSO2 Identity Server are separated into two general components, as shown below. You can reference these components when you write the custom layout using a special notation.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md (hunk 5 above)
Comment (Yoshani) [reply en/docs/references/extend/rebranding/customizable-login-portal.md:3]: `username and password recovery pages` is one item

**Review by divyaamunugama — APPROVED** [review]

**Pull request merged**